MyHome.DataRepositories/GeneralCategoryHandler.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using MyHome.Persistence;
+
+namespace MyHome.DataRepository
+{
+    // TODO - QC - Create a service that merges the categories
```

Comment:
I dislike using a string to retrieve a certain type of data, maybe the ui should be less "Dynamic" and we should use the type service directly.